File: dataModule/src/main/java/com/mobnetic/coinguardian/util/ParseUtils.java
package com.mobnetic.coinguardian.util;

import org.json.JSONException;
import org.json.JSONObject;

public class ParseUtils {

	public static double getDoubleFromString(JSONObject jsonObject, String name) throws NumberFormatException, JSONException {
		return Double.parseDouble(jsonObject.getString(name));
	}
}

File: dataModule/src/main/java/com/aneonex/bitcoinchecker/datamodule/util/ParseUtils.kt
package com.aneonex.bitcoinchecker.datamodule.util

import org.json.JSONException
import org.json.JSONObject

object ParseUtils {
    @Throws(NumberFormatException::class, JSONException::class)
    fun getDoubleFromString(jsonObject: JSONObject, name: String): Double {
        return jsonObject.getString(name).toDouble()
    }
}